deployment/geotrellis-cluster.config.example

```diff
@@ -0,0 +1,13 @@
+[CONFIG NAME]
+IPAccess: '<IP to allow SSH Access> (e.g. 216.158.51.82/32)'
+KeyName: '<EC2 Key to authenticate with SSH>'
+MesosFollowerAMI: '<AMI ID of Mesos Follower (optional -- will be found automatically if not provided)>'
```

Comment:
I was trying to keep them alphabetical to make it easier to find where they are, logically I agree though it probably makes sense to put the leader stuff first